game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts

```diff
@@ -103,17 +103,16 @@ function onStart(complete: () => void) {
                 }
             }),
             tg.wait(0.5),
-
+            tg.immediate(_ => playerHero.SetMoveCapability(UnitMoveCapability.GROUND)),
             tg.immediate(context => {
                 const riki = getOrError(context[CustomNpcKeys.Riki] as CDOTA_BaseNPC | undefined);
                 const tricksOfTheTrade = riki.GetAbilityByIndex(2);
                 if (tricksOfTheTrade) {
                     riki.CastAbilityOnPosition(riki.GetAbsOrigin().__add(Vector(-200, 100)), tricksOfTheTrade, 0);
                 }
             }),
-            tg.wait(3),
-
-            tg.immediate(_ => playerHero.SetMoveCapability(UnitMoveCapability.GROUND)),
+            tg.audioDialog(LocalizationKey.Script_4_RTZ_getaway, LocalizationKey.Script_4_RTZ_getaway, ctx => ctx[rikiName]),
+            tg.wait(2),
```

Comment:
Replace the wait with the audioDialog's last argument for the extra delay.

Reply:
replaced.